frontend/js/main.js

```diff
@@ -0,0 +1,1910 @@
+/**
+ * Function to initiate home screen tasks like validating form input fields
+ *
+ * @return {null}
+ */
+const initHomeScreenTasks = () => {
+  jQuery(document).ready(function () {
+    setActiveSelectedMenu('homeScreen');
+    jQuery('#loadDbForm > div > input').keyup(function () {
```

Comment:
I need a working version of this to test this, but I'm not sure if this works for the paste case (right click -> paste). I don't think that triggers the keyup function. Not too sure. If it doesn't trigger the keyup case, then the button will still be disabled even when there are contents pasted.

Reply:
Currently there are few dependencies from backend code to merge in master branch therefore working version is not available to test this.
- Checked paste case scenario (right click -> paste) and it is triggering the keyup function